inc/autoloader.php

```diff
@@ -55,7 +55,11 @@ public function autoload( $class ) {
 			if ( $dir['suffix'] && strlen( $file ) > strlen( $dir['suffix'] ) ) {
 				$file = substr( $file, 0, - strlen( $dir['suffix'] ) );
 			}
-			$file = strtolower( str_replace( '_', '-', $file ) ) . '.php';
+			if ( function_exists( 'mb_strtolower' ) && function_exists( 'mb_strtolower' )  ) {
+				$file = mb_strtolower( str_replace( '_', '-', $file ), mb_detect_encoding( $file ) ) . '.php'
```

Comment:
I think using either using one condition for `mb_strtolower` or `function_exists( 'mb_detect_encoding' )`. Currently, both checks are for the same function.

Reply:
Ha, too tired. After 1am here. I’ll fix in the morning. ;)